src/main/java/de/tum/in/www1/artemis/service/quiz/QuizExerciseImportService.java

```diff
@@ -71,10 +75,11 @@ public QuizExerciseImportService(QuizExerciseService quizExerciseService, FileSe
      *
      * @param templateExercise The template exercise which should get imported
      * @param importedExercise The new exercise already containing values which should not get copied, i.e. overwritten
+     * @param files            The potential files to be added. Null if no change to files during import. ExamImportService sends null by default
      * @return The newly created exercise
      */
     @NotNull
-    public QuizExercise importQuizExercise(final QuizExercise templateExercise, QuizExercise importedExercise) {
+    public QuizExercise importQuizExercise(final QuizExercise templateExercise, QuizExercise importedExercise, @Nullable List<MultipartFile> files) throws IOException {
```

Comment:
Is this just affecting new files, i.e. existing files won't be changed?

If yes, I would probably overload the method as setting it to null essentially means "no additional effect".

Reply:
I get what you mean, but this method is only used twice and I dont think there is any clear benefit in this instance to overload the method.